docs/source/python_api/mlflow.entities.rst

```diff
@@ -4,4 +4,3 @@ mlflow.entities
 .. automodule:: mlflow.entities
     :members:
     :undoc-members:
-    :show-inheritance:
```

Comment:
Removing `show-inheritance` here because the classes in this module inherit from the private `_MLflowObject` class, which is not included in the docs rendered by Sphinx. The current build process produces broken links to nonexistent `_MLflowObject` documentation.